pandas/tests/frame/test_analytics.py

```diff
@@ -1031,6 +1031,34 @@ def test_idxmax(self):
 
         pytest.raises(ValueError, frame.idxmax, axis=2)
 
+    def test_argmin(self):
+        frame = self.frame
+        frame.loc[5:10] = np.nan
+        frame.loc[15:20, -2:] = np.nan
+        for skipna in [True, False]:
+            for axis in [0, 1]:
+                for df in [frame, self.intframe]:
+                    result = df.argmin(axis=axis, skipna=skipna)
+                    expected = df.apply(Series.argmin, axis=axis,
+                                        skipna=skipna)
+                    tm.assert_series_equal(result, expected)
+
+        pytest.raises(ValueError, frame.argmin, axis=2)
+
+    def test_argmax(self):
```

Comment:
No, he's referring to the fact that you were using nested for-loops for testing.  You can avoid those by parametrizing as illustrated <a href="https://docs.pytest.org/en/latest/parametrize.html">here</a>